### PR TITLE
Added error handling for print my barcode errors.

### PR DESCRIPTION
**Pull request by stevieing**
Closes #

#### Changes proposed in this pull request

- 

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

**Review by dasunpubudumal — DISMISSED** [review]

File: src/api/v2/ResponseHelper.js [hunk 1]
```diff
@@ -39,20 +34,51 @@ const parseErrorObject = (errors) => {
...
- * @param {Object} e.g. { success = true, body: { data: { id: 1 } }, errors = { error1: ['nasty'], error2: ['broken', 'crushed'], }, error = 'there was an error', error = { title: 'Invalid field', detail: 'tag_group is not a valid includable relationship of tags', code: '112', status: '400',
+ * @param {Array} errors
+ * @returns {String} - comma separated string of key, value pairs
+ * // e.g.  errors: [{ source: { pointer: '/data/attributes/printer', }, detail: "can't be blank", },] would return 'printer can't be blank'
+ */
+const parsePrintMyBarcodeErrors = (errors) => {
+  return errors
+    .map(({ source, detail }) => `${source.pointer.split('/').pop()} ${detail}`)
+    .join(', ')
+}
```
Comment (seenanair) [inline src/api/v2/ResponseHelper.js:49]: Is there tests for this function?

**Review by seenanair — COMMENTED** [review]
Looks good 👍 
Is parsePrintMyBarcodeErrors
One doubt, so in real situation where are we placing or determining the error handler type is printMyBarcodeErrors?

File: src/api/v2/ResponseHelper.js (hunk 1 above)
Comment (stevieing) [reply src/api/v2/ResponseHelper.js:49]: Yes. In ResponseHelper.spec.js 'failure with print my barcode request'

**Review by seenanair — DISMISSED** [review]

**Comment by stevieing** [comment]
> Looks good 👍 Is parsePrintMyBarcodeErrors One doubt, so in real situation where are we placing or determining the error handler type is printMyBarcodeErrors?

It will work by passing in the errorHandler to handleResponse. It looks like I have made a mistake and not added that so good spot and I will change it now.

**Review by seenanair — APPROVED** [review]

**Review by dasunpubudumal — APPROVED** [review]

**Pull request merged**